src/Components/TestExplorer.fs

```diff
@@ -94,10 +94,21 @@ module CancellationToken =
 
         tokenSource.token
 
+
+
 type TestId = string
 type ProjectPath = string
 type TargetFramework = string
 
+module Project =
+    let testPathSubDir = ".ionide-test"
+    let getOutputPaths () =
+        let objOutputPath = node.path.join [|  "obj" ; testPathSubDir  ; node.path.sep|]
```

Comment:
Using relative paths means it can build each respective project in its own subfolder (Real lib and test project), otherwise, things like project.assets.json will collide. . Though I'm not entirely sure this is the correct approach.

Reply:
using separate subfolders is the right move here IMO

Reply:
This is the same discussion that's going on for the `dotnet run foo.cs` work, and separate intermediate/output paths is what that effort has landed on as well.